simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderColorSchemes.java

```diff
@@ -26,9 +26,9 @@ public static int background(final ReaderColorScheme scheme)
       case SCHEME_BLACK_ON_BEIGE:
         return Color.argb(0xff, 242, 228, 203);
       case SCHEME_BLACK_ON_WHITE:
-        return Color.WHITE;
+        return 0xff000000 | (Color.WHITE & 0xffffff);
```

Comment:
What does the `& 0xffffff` do?

Reply:
The values returned from this method have to be packed AARRGGBB integers to be consumed by the Android API. I think some of the constants are defined with 0 in the alpha channel, so that expression masks off the RGB bits and then combines them with a value that represents a fully-opaque alpha channel. This was necessary to avoid the header and footer in the reader appearing to be black.